app/app.cpp

```diff
@@ -1,11 +1,49 @@
 #include "app.h"
+#include <sstream>
+#include <iomanip>
 #include <format>
 
 namespace vsite::oop::v1
 {
-/*
-* function implementations
-*/
+    std::string to_hex(int value) {
+        std::stringstream ss;
```

Comment:
Ne, koristite [std::format](https://en.cppreference.com/w/cpp/utility/format/format), kao što smo pokazivali na predavanjima.